tests/nnstreamer_filter_mvncsdk2/unittest_filter_mvncsdk2.cc

```diff
@@ -48,7 +48,10 @@ TEST (pipeline_mvncsdk2_filter, launch_normal)
   if (gstpipe) {
     GstState state;
     GstStateChangeReturn ret;
+    GstElement *filter;
```

Comment:
`filter` is not unreffed with `gst_object_unref()`. is it ok?

Reply:
My bad.. Thank you! Updated.